main.js

```diff
@@ -0,0 +1,280 @@
+// const axios = require('axios')
+
+let countries = [];
+let APIerror = "";
```

Comment:
Deze zou ik niet globaal definiëren, maar pas initialiseren in de functies die hier wat mee doen

Reply:
APIerror wordt in verschillende andere functies afgetest op inhoud. De testen zouden dan ook gewijzigd moeten worden.
Commentaar toegevoegd bij de definitie, om dit duidelijk te maken.